pallets/broadcast/src/lib.rs

```diff
@@ -151,7 +153,9 @@ impl<T: Config> Pallet<T> {
 		Ok(next_id)
 	}
 
-	pub fn remove_from_context() -> DispatchResult {
+	pub fn reset_context() -> DispatchResult {
```

Comment:
i dont get the rename, it sound to me like you are cleaning everything, but in fact you are just popping one stack level same as before, therefore remove_from_context is better as mirror to the add_to_context which adds one level

Reply:
we are clearing everything, so all the broadcast storage, including stack and swapper, hence the renaming.

Reply:
you are not clearing everything, just removing one level from stack

Reply:
you are right, reset is indeed a wrong term, reverted these changes back